### ISSUE TITLE
Chaining modals (queue) not working on v6.4.3

### ISSUE THREAD

**Issue by SRWieZ** [limonte/sweetalert2#469]
Not sur why but it work's on **v6.4.2** but not in **v6.4.3**

Not that I'm not using `input: 'url'`

To reproduce the bug, juste use **Chaining modals (queue) example** on https://limonte.github.io/sweetalert2/

Have a good day !

**Comment by limonte** [comment]
I can't reproduce.  Which browser are you using? 

Can anyone else confirm the issue?

**Comment by toverux** [comment]
No problem for me under Chromium and Firefox.
@SRWieZ please tell us what browser + version you are using, and if you encounter errors in the JavaScript console.

**Comment by SRWieZ** [comment]
Firefox 52.0

Sorry, I didn't mention that the modal appear but it's stuck at the first step.
No errors in console.

**Comment by birjolaxew** [comment]
Could you please try starting Firefox in safe mode (hold <kbd>Shift</kbd> while launching it), and testing again?

**Comment by SRWieZ** [comment]
Same.

To be sure, I tested it outside my project.
```
<script src="vendor/jquery/jquery.min.js"></script>
<script src="vendor/es6-promise/es6-promise.auto.min.js"></script>
<script src="vendor/sweetalert2/sweetalert2.min.js"></script>
<link rel="stylesheet" href="vendor/sweetalert2/sweetalert2.min.css"/>
```
jQuery v3.1.0

**Comment by marekmaurizio** [comment]
Same problem.

**Comment by birjolaxew** [comment]
[I still can't reproduce on my end.](http://i.imgur.com/dfieAjb.gifv) Can you verify that you're experiencing the issue on [this JSFiddle](https://jsfiddle.net/fhkyfsdn/)? I'm not really sure what to do if you are; if so, please join [our Gitter](https://gitter.im/sweetalert2/Lobby) for more real time debugging.

**Comment by marekmaurizio** [comment]
The fiddle works for me. 
I can confirm however that my code works if i downgrade to 4.6.2.

**Comment by SRWieZ** [comment]
The fiddle works for me too.
Really weird.

Can you try with `dist/sweetalert2.min.js` of https://github.com/limonte/sweetalert2/archive/v6.4.3.zip

Work's with `sweetalert2.js` but not with `sweetalert2.min.js`

**Comment by birjolaxew** [comment]
If one of you could please join [our Gitter](https://gitter.im/sweetalert2/Lobby) so we can debug this, that'd be great.

**Comment by agauniyal** [comment]
Hi I'm having this issue as well, latest Chrome on Linux. The modal is stuck on first chained step.

**Comment by limonte** [comment]
Fixed in [v6.4.4](https://github.com/limonte/sweetalert2/releases/tag/v6.4.4)

Thanks all for reporting and confirming this weird issue with minified version in `6.4.3`.
